lib/caracal/renderers/header_renderer.rb

```diff
@@ -0,0 +1,62 @@
+require 'nokogiri'
+
+require 'caracal/renderers/xml_renderer'
+
+
+module Caracal
+  module Renderers
+    class HeaderRenderer < DocumentRenderer
+
+      #-------------------------------------------------------------
+      # Public Methods
+      #-------------------------------------------------------------
+
+      # This method produces the xml required for the `word/settings.xml`
```

Comment:
This comment isn't valid for this file, right?  This produces xml require for `word/header1.xml`?

Reply:
Ah, thanks for catching that! Looks like there is a similar issue with the footer renderer.